test/integration/proportion-boolean/cql/proportion-boolean.cql

```diff
@@ -0,0 +1,14 @@
+library ProportionBooleanTest version '0.0.1'
+
+using FHIR version '4.0.1'
+
+include FHIRHelpers version '4.0.1'
+
```

Comment:
```suggestion
parameter "Measurement Period" Interval<DateTime>
  default Interval[@2019-01-01T00:00:00.0, @2020-01-01T00:00:00.0)

context Patient

```

This example should have the MP and needs `context Patient`

Reply:
Good call! Gonna make the default 2022